config/crds/hive_v1alpha1_clusterstate.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: clusterstates.hive.openshift.io
+spec:
+  group: hive.openshift.io
+  names:
+    kind: ClusterState
+    plural: clusterstates
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
```

Comment:
I understand that these are mostly copied from standard templates, but these links `https://git.k8s.io/community/contributors/devel/api-conventions.md#resources` does not exist right now. Not sure what is the correct thing to do.

Reply:
these aren't copied, these are generated by the controller-tools crd generation tool. We should likely update to a more recent version.

Reply:
> e should likely update to a more recent version.



> We should likely update to a more recent version.

Can we have a Jira for this? You can assign this to me.

Reply:
https://jira.coreos.com/browse/CO-535